constants.py

```diff
@@ -1,5 +1,7 @@
 from enum import Enum
 
+INFINITY = 1000;
```

Comment:
FYI, you could set `INFINITY` to a constant like `float('inf')` and checking explicitly for that.

https://stackoverflow.com/questions/34264710/what-is-the-point-of-floatinf-in-python